helm-charts/submarine/Chart.yaml

```diff
@@ -23,6 +23,10 @@ version: 0.8.0-SNAPSHOT
 icon: https://submarine.apache.org/assets/themes/submarine/img/submarine_white_logo.png
 dependencies:
   - name: training-operator
-    version: "0.1.0"
+    version: "0.7.0"
   - name: notebook-controller
     version: "0.1.0"
+  - name: seldon-core-operator
```

Comment:
I think we should add the `helm dep update` to https://github.com/apache/submarine/blob/master/website/docs/gettingStarted/quickstart.md and add the dependency description.